Makefile

```diff
@@ -213,7 +213,7 @@ GIT_VERSION: .git
 docker-cilium-image-for-developers:
 	# DOCKER_BUILDKIT allows for faster build as well as the ability to use
 	# a dedicated dockerignore file per Dockerfile.
-	DOCKER_BUILDKIT=1 $(CONTAINER_ENGINE_FULL) build \
+	DOCKER_BUILDKIT=1 $(CONTAINER_ENGINE) build \
```

Comment:
maybe this one can get `$(QUIET)` (at the beginning of the line) too?

Reply:
I was wondering what I should do with this one. I was not sure how `@` interacts with environment variables. But now I have an answer that mixing those is fine. ;)